org.scala-ide.sdt.debug/src/org/scalaide/debug/internal/model/ScalaDebugTarget.scala

```diff
@@ -158,8 +166,14 @@ abstract class ScalaDebugTarget private(val virtualMachine: VirtualMachine,
   @volatile
   private var threads = List[ScalaThread]()
 
+  @volatile
+  private var isDuringHcr: Boolean = false
+  private[internal] def isPerformingHotCodeReplace: Boolean = isDuringHcr
+  private[internal] def isPerformingHotCodeReplace_=(isPerforming: Boolean) = { isDuringHcr = isPerforming }
```

Comment:
I'm not sure what is the difference between having a `var` and having a getter *and* a setter for a private var. In this case it seems there is none.